src/fragments/products.ts

```diff
@@ -110,6 +110,14 @@ export const productFragment = gql`
       name
       hasVariants
     }
+    category {
+      id
+      name
+    }
+    collections {
+      id
+      name
+    }
```

Comment:
We can use `@include(if: ...)` statements when the columns are toggled, otherwise we download the data even if we don't use them, especially when they are disabled by default

Reply:
405b68c